### PR TITLE
style(CompileMetadataResolver): better error message

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

File: modules/@angular/compiler/src/metadata_resolver.ts [hunk 1]
```diff
@@ -307,7 +307,7 @@ export class CompileMetadataResolver {
         const typeMetadata = flattenArray(meta.bootstrap).map(type => {
           if (!isValidType(type)) {
             throw new Error(
-                `Unexpected value '${stringify(type)}' used in the bootstrap property of module '${stringify(moduleType)}'`);
+                `Unexpected value '${stringify(type)}' used in the bootstrap property of module '${stringify(moduleType)}'`);p
```
Comment (vicb) [inline modules/@angular/compiler/src/metadata_resolver.ts]: ;p

File: modules/@angular/compiler/src/metadata_resolver.ts [hunk 2]
```diff
@@ -377,7 +377,7 @@ export class CompileMetadataResolver {
     const oldModule = this._ngModuleOfTypes.get(type);
     if (oldModule && oldModule !== moduleType) {
       throw new Error(
-          `Type ${stringify(type)} is part of the declarations of 2 modules: ${stringify(oldModule)} and ${stringify(moduleType)}!`);
+          `Type ${stringify(type)} is part of the declarations of 2 modules: ${stringify(oldModule)} and ${stringify(moduleType)}! Please consider moving ${stringify(type)} to a higher module that imports ${stringify(oldModule)} and ${stringify(moduleType)}. You can also create a new NgModule that exports and includes ${stringify(type)} then import that NgModule in ${stringify(oldModule)} and ${stringify(moduleType)}.`);
```
Comment (vicb) [inline modules/@angular/compiler/src/metadata_resolver.ts]: Please consider splitting the msg over multiple lines & squash the commits. Thanks

**Comment by gdi2290** [comment]
@vicb all green

**Comment by vicb** [comment]
thanks Patrick

**Pull request merged**